repo/packages/C/chronos/0/marathon.json

```diff
@@ -28,6 +28,8 @@
       "portIndex": 0,
       "protocol": "HTTP"
     }
-  ]
-
+  ],
+  "labels": {
+    "DCOS_PACKAGE_FRAMEWORK_NAME": "{{chronos/mesos-framework-name}}"
```

Comment:
We had an internal JIRA for how this was supposed to work (DCOS-774).  It's been closed, maybe @jsancio has more details about decisions that were made after I stopped tracking it :)